### PR TITLE
✨ Support login with WalletConnect

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

**Review by williamchong — APPROVED** [review]
Strange that the store is called `keplr` but has walletconnect codes, maybe rename to wallet?

**Comment by nwingt** [comment]
@williamchong Renamed

File: constant/index.ts [hunk 1]
```diff
@@ -54,3 +54,8 @@ export const WALLET_TYPES = [
 ]
 
 export const SITE_URL = IS_TESTNET ? 'https://app.rinkeby.like.co' : 'https://app.like.co';
+
+export const CONNECT_WALLET_TYPES = [
+  'keplr',
+  'likerId',
```
Comment (williamchong) [inline constant/index.ts:60]: should this be named `likerland` as in `likerland app`

**Review by williamchong — CHANGES_REQUESTED** [review]

File: constant/index.ts (hunk 1 above)
Comment (nwingt) [reply constant/index.ts:60]: How about `likerland_app`?

Comment (williamchong) [reply constant/index.ts:60]: sure

**Review by williamchong — APPROVED** [review]

**Pull request merged**